### ISSUE TITLE
System.AggregateException: A Task's exception(s) were not observed

### ISSUE THREAD

**Issue by Firemad03** [Radarr/Radarr#2891]
I saw in Radarr this error and I needed to restart it to make it work again.

Version: 0.2.0.1123

System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. ---> System.Net.WebException: The request was aborted: The request was canceled.
  at System.Net.HttpWebRequest+<MyGetResponseAsync>d__246.MoveNext () [0x003d5] in <fc308f916aec4e4283e0c1d4b761760a>:0 
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.Net.WebException: The request was aborted: The request was canceled.
  at System.Net.HttpWebRequest+<MyGetResponseAsync>d__246.MoveNext () [0x003d5] in <fc308f916aec4e4283e0c1d4b761760a>:0 <---

**Comment by galli-leo** [comment]
Duplicate of #2904
